src/phoenix/server/api/subscriptions.py

```diff
@@ -64,36 +86,79 @@ async def chat_completion(
 
         client = AsyncOpenAI()
 
-        # Loop over the input messages and map them to the OpenAI format
-
-        messages: List[ChatCompletionMessageParam] = [
-            to_openai_chat_completion_param(message) for message in input.messages
-        ]
-        chunk_contents = []
-        start_time = datetime.now()
-        async for chunk in await client.chat.completions.create(
-            messages=messages,
-            model="gpt-4",
-            stream=True,
-        ):
-            choice = chunk.choices[0]
-            if choice.finish_reason is None:
-                assert isinstance(chunk_content := chunk.choices[0].delta.content, str)
-                yield chunk_content
-                chunk_contents.append(chunk_content)
-        end_time = datetime.now()
-        async with info.context.db() as session:
-            # insert dummy data
-            trace_id = str(start_time)
-            span_id = str(end_time)
-            default_project_id = await session.scalar(
-                select(models.Project.id).where(models.Project.name == "default")
+        in_memory_span_exporter = InMemorySpanExporter()
+        tracer_provider = TracerProvider()
+        tracer_provider.add_span_processor(
+            span_processor=SimpleSpanProcessor(span_exporter=in_memory_span_exporter)
+        )
+        tracer = tracer_provider.get_tracer(__name__)
+        span_name = "ChatCompletion"
+        with tracer.start_span(
```

Comment:
what's the reason we need an otel span? it seems extraneous in this context, since it just gets discarded after all

Reply:
we don't have to use otel. i just wanted to make sure those details like exception messages, etc., where consistent, but it's not 100% necessary